subprojects/stc.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = STC-5.0
+source_url = https://github.com/stclib/STC/archive/refs/tags/v5.0.tar.gz
+source_filename = v5.0.tar.gz
+source_hash = d61353852b9d7ef69b56fa32edcbc7934f2153385f3778536234201ceebcc950
+
+[provide]
+dependency_names = stc
```

Comment:
Interestingly, it appears they provide a `declare_dependency()` but not an installable `import('pkgconfig').generate(stc_lib)`.

Reply:
Thanks for the catch, I'll add it.